servers/src/main/java/tachyon/worker/block/meta/StorageDir.java

```diff
@@ -60,6 +62,48 @@ public StorageDir(StorageTier tier, int dirIndex, long capacityBytes, String dir
     mBlockIdToBlockMap = new HashMap<Long, BlockMeta>(200);
     mBlockIdToTempBlockMap = new HashMap<Long, TempBlockMeta>(200);
     mUserIdToTempBlockIdsMap = new HashMap<Long, Set<Long>>(200);
+
+    initializeMeta();
```

Comment:
In general, calling a member method inside a class's constructor is risky and should be avoided, or at least put careful documentation. Maybe leave a TODO here so we can come back later to improve the code: e.g., we can make a factory method to create `StorageDir` and move `inializeMeta` into the factory method. One advantage is that, it is more appropriate for a factory method to throw exceptions.